huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_partition_test.go

```diff
@@ -0,0 +1,139 @@
+package cce
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk/openstack/cce/v3/partitions"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func getPartitionResourceFunc(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	c, err := conf.CceV1Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating CCE v1 client: %s", err)
+	}
+	resp, err := partitions.Get(c, state.Primary.Attributes["cluster_id"],
+		state.Primary.Attributes["availability_zone"]).Extract()
+	if resp == nil && err == nil {
+		return resp, fmt.Errorf("unable to find the partition (%s)", state.Primary.ID)
+	}
+	return resp, err
+}
+
+func TestAccCCEPartition_basic(t *testing.T) {
+	var partition partitions.Partitions
+	resourceName := "huaweicloud_cce_partition.test"
+	randName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	// The availability zone of IES edge partition, example: "cn-south-1-ies-fstxz"
+	azName := acceptance.HW_CCE_PARTITION_AZ
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&partition,
+		getPartitionResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheckCcePartitionAz(t)
```

Comment:
```suggestion
			acceptance.TestAccPreCheck(t)
			acceptance.TestAccPreCheckCcePartitionAz(t)
```
the basic check is missing

Reply:
OK